### ISSUE TITLE
Can leblebi make suggestions based on prefixes?

### ISSUE THREAD

**Issue by hrzafer** [ahmetaa/leblebi#2]
Leblebi can make suggestions for complete words such as "tubitak => tübitak" or "besiktas => beşiktaş". But is a suggestion like "tubi => tübi" possible?

**Comment by ahmetaa** [comment]
No, that is not possible now. However, I believe such a thing can be added. Better is to provide a real suggestion mechanism which does not exist yet.
